constants/constants.go

```diff
@@ -8,6 +8,11 @@ const CustomDateFormat = "2006-01-02"
 // ActionItemType is datatype for ActionItems's
 type ActionItemType string
 
+// constats to be used for task providers
+const (
+	PivotalTracker = "pivotal"
```

Comment:
Please, replace `"pivotal"` string from the whole code by this constant.

Reply:
done